src/pages/RunDetails.test.js

```diff
@@ -20,7 +20,7 @@ const runDetailsProps = {
   namespace: 'example-billing-project',
   name: 'workspace',
   submissionId: 1,
-  workflowId: 2
+  workflowId: '5d96fd3c-1a89-40ae-8095-c364181cda46'
```

Comment:
I once lost several hours trying to find the origin of a real-looking but hardcoded workflow ID and now believe in fake-looking IDs
```suggestion
  workflowId: '00001111-2222-3333-aaaa-bbbbccccdddd'
```

Reply:
Totally 👍 This needs to be updated in two places, so will push a separate commit.